tests/audit-tests/null/result.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Again, to be removed

Reply:
Same as above. These files come back after every run